platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/ConnectorServices.java

```diff
@@ -57,6 +57,7 @@ public static void reset() {
         singletonRegistry = new ConnectorServiceRegistry();
     }
 
+    @SuppressWarnings("UnusedMethod")
```

Comment:
Is it possible to put this annotation on the offending methods instead of the class?
It's usually better to avoid new unused methods creeping into the same class. Also, one would have to remove the suppression to understand what methods are actually unused.